src/corefx/System.Globalization.Native/pal_icushim.c

```diff
@@ -78,77 +78,77 @@ void GetVersionedLibFileName(const char* baseFileName, int majorVer, int minorVe
         if (subVer != -1)
         {
             sprintf(result + nameLen, ".%d", subVer);
-        }    
+        }
     }
 }
 
-bool FindSymbolVersion(int majorVer, int minorVer, int subVer, char* symbolName, char* symbolVersion)
+static int FindSymbolVersion(int majorVer, int minorVer, int subVer, char* symbolName, char* symbolVersion)
 {
     // Find out the format of the version string added to each symbol
     // First try just the unversioned symbol
-    if (dlsym(libicuuc, "u_strlen") == nullptr)
+    if (dlsym(libicuuc, "u_strlen") == NULL)
     {
         // Now try just the _majorVer added
         sprintf(symbolVersion, "_%d", majorVer);
         sprintf(symbolName, "u_strlen%s", symbolVersion);
-        if ((dlsym(libicuuc, symbolName) == nullptr) && (minorVer != -1))
+        if ((dlsym(libicuuc, symbolName) == NULL) && (minorVer != -1))
         {
             // Now try the _majorVer_minorVer added
             sprintf(symbolVersion, "_%d_%d", majorVer, minorVer);
             sprintf(symbolName, "u_strlen%s", symbolVersion);
-            if ((dlsym(libicuuc, symbolName) == nullptr) && (subVer != -1))
+            if ((dlsym(libicuuc, symbolName) == NULL) && (subVer != -1))
             {
                 // Finally, try the _majorVer_minorVer_subVer added
                 sprintf(symbolVersion, "_%d_%d_%d", majorVer, minorVer, subVer);
                 sprintf(symbolName, "u_strlen%s", symbolVersion);
-                if (dlsym(libicuuc, symbolName) == nullptr)
+                if (dlsym(libicuuc, symbolName) == NULL)
                 {
-                    return false;
+                    return FALSE;
```

Comment:
C99 `<stdbool.h>` is used in multiple places in CoreFX [ref](https://github.com/dotnet/corefx/search?q=stdbool), could it be used instead to save 3 bytes per instance, per call?